src/hammer-vlsi/par/openroad/__init__.py

```diff
@@ -351,6 +351,9 @@ def init_design(self) -> bool:
         # set up useful variables
         clock_port = self.get_clock_ports()[0]
         self.clock_port_name = clock_port.name
+        tech_base_dir=self.get_setting('vlsi.core.technology_path')[0]
```

Comment:
This should probably be indexed to [-1]. Oftentimes for tech plugins not in Hammer, the `technology_path` key is `_meta: append` -ed to.